interview-questions.md

```diff
@@ -7,60 +7,58 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a function? Why would you use one?
 
-  Your answer:
-
-  Researched answer:
-
-
+  Your answer: A function is a chunk of reusable code that perfoms some logic to solve a either a reoccuring problem or a simplfy a task. You would use a function to perform a task that you want your code to complete or do.
 
+  Researched answer: From *'MDN Web Docs,'* webiste a functions is similiar to percedure with a set of statements that performs a taske or calculates a value. However, in order for it to quaily as a function it should take some input and return an output.
+  
 2. In regards to functions, what is the difference between an argument and a parameter?
 
-  Your answer:
+  Your answer: A parameter is a placeholder for the value within the functions definition, while the argument is the actual value being passed in to the function.
 
-  Researched answer:
+  Researched answer: From *'MDN Web Docs,'* A parameter is the name listed in the function's definition and the arguements are the real values that are passed to the function. It then describes the actions between the two associations, so when a parameter is initialized, it is initialized to the vales of the arguements that are supplied. 
```

Comment:
Look at you pulling an Austin. It's different enough I'll let it slide... this time